scss/mixins/_focus.scss

```diff
@@ -0,0 +1,15 @@
+// scss-docs-start focus-visible
+@mixin focus-visible($zindex: $outline-zindex, $color: $outline-color, $width: $outline-width, $offset: $outline-offset) {
+  z-index: $zindex;
+  outline: $color solid $width;
+  outline-offset: $offset;
+}
+// scss-docs-end focus-visible
+
+// scss-docs-start focus-box
+@mixin focus-box($offset: $outline-offset, $box-color: $outline-box-color) {
+  @if $enable-shadows {
```

Comment:
This `@if $enable-shadows {` condition should be deleted in order to have a white circle between the carousel left/right arrow and its focus. Nevertheless, this deletion might break things somewhere else => to be investigated before deleting.